scripts/src/models/orders.ts

```diff
@@ -329,6 +334,14 @@ export type P2POrder = Order & {
 	contexts: [OrderContext, OrderContext];
 };
 
+export type CrossAppOrder = Order & {
+	sender: User;
+	recipient: User;
```

Comment:
currently you will only have the sender side of the CrossApp, but when you implement the receiver side you'll have always only one of them. Maybe use Typescript's `{sender: User} | {recipient: User}`? Or just like `NormalOrder`, just have `{user: User}`

Reply:
It was copied from P2POrder... yet I found out I don't really need this type so removed it altogether